go.mod

```diff
@@ -293,12 +293,10 @@ require (
 )
 
 replace (
-	// branch: v0.28.0x-osmo-v12, current tag: v0.28.0-osmo-v12.1
-	github.com/CosmWasm/wasmd => github.com/osmosis-labs/wasmd v0.28.0-osmo-v12.1
-	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current branch: osmosis-main. Direct commit link: https://github.com/osmosis-labs/cosmos-sdk/commit/2e4ae2f7619a
-	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.45.1-0.20221004142537-2e4ae2f7619a
-	// Use Osmosis fast iavl
-	github.com/cosmos/iavl => github.com/osmosis-labs/iavl v0.17.3-osmo-v7
+	// branch: v0.28.0x-osmo-v13, current tag: v0.28.0-osmo-v13
+	github.com/CosmWasm/wasmd => github.com/osmosis-labs/wasmd v0.28.0-osmo-v13
+	// Our cosmos-sdk branch is:  https://github.com/osmosis-labs/cosmos-sdk, current branch: osmosis-main. Direct commit link: https://github.com/osmosis-labs/cosmos-sdk/commit/c1eb80426bbcbd691c4522128840264a5c695e42
```

Comment:
Jut a note that for the final v13, we should remember to cut an SDK `v13.x` and take a tag from it.